src/seedsigner/helpers/embit_utils.py

```diff
@@ -200,3 +201,31 @@ def sign_message(seed_bytes: bytes, derivation: str, msg: bytes, compressed: boo
     flag = bytes([27 + flag + c])
     ser = flag + secp256k1.ecdsa_signature_serialize_compact(sig._sig)
     return b2a_base64(ser).strip().decode()
+
+
+
+# Basic BIP-352 Silent Payments support
+def encode_silent_payment_address(scan_privkey: ec.PrivateKey, spend_pubkey: ec.PublicKey, label: int | str | bytes | None = None, network: str = "main", version: int = 0) -> str:
+    """
+    Adapted from https://github.com/bitcoin/bips/blob/master/bip-0352/reference.py
+
+    Generates the recipient's reusable silent payment address for a given:
+        * scanning key (can be PublicKey or PrivateKey)
+        * spending pubkey
+        * optional label for labeled addresses
+
+    if scan_key is a PrivateKey and label is provided, generates a labeled address.
+    """
```

Comment:
Maybe add a `TODO` note that we'd like to eventually see this functionality moved into `embit`.